### DISCUSSION TITLE
Ability to add comments under beatmaps in osu!lazer

### DISCUSSION THREAD

**Discussion by Coldtrigon66** [ppy/osu#17515] [Ideas] [answered]
I like how the website has been coded into lazer, tho the ability to comment under beatmaps seems to be missing

**Comment by bdach** [comment, marked as answer]
Sort of in scope of https://github.com/ppy/osu/issues/7013 (although that issue is a bit on the large side)

**Reply by Coldtrigon66** [reply]
Thanks
